src/installing-workflow/system-requirements.md

```diff
@@ -24,11 +24,11 @@ Deis workflow has been tested with the Kubernetes v1.2 release line. It is incom
 
 ## Docker Version
 
-Any Kubernetes 1.1 cluster should also use a Docker version < 1.10.0 so that `kubectl exec` and Deis database health checks work properly (Refs: [fsouza/go-dockerclient#455](https://github.com/fsouza/go-dockerclient/issues/455) and [kubernetes/kubernetes#19720](https://github.com/kubernetes/kubernetes/issues/19720)).
+Any Kubernetes 1.1 cluster should also use a Docker version before 1.10.0 so that `kubectl exec` and Deis database health checks work properly (Refs: [fsouza/go-dockerclient#455](https://github.com/fsouza/go-dockerclient/issues/455) and [kubernetes/kubernetes#19720](https://github.com/kubernetes/kubernetes/issues/19720)).
```

Comment:
I think this restriction has changed if we're talking about k8s 1.2+ only, which I believe at this point we should be.